frontend/better-you/src/redux/reducers/appHandlers.ts

```diff
@@ -496,4 +496,25 @@ export function addFriendErrorHandler(oldState: AppState, error: string): AppSta
   newState.error = error;
   newState.challenges = [];
   return newState;
+} 
+
+export function fetchFriendGoalsBeginHandler(oldState: AppState): AppState {
+  const newState = {...oldState};
+  newState.loading = true;
+  newState.error = "";
+  return newState;
+}
+export function fetchFriendGoalsSuccessHandler(oldState: AppState, goals: Goal[]): AppState {
+  const newState = {...oldState};
+  newState.loading = false;
+  newState.error = "";
+  newState.goals = goals;
```

Comment:
newState.goals sunt goal-urile user-ului logat, nu ai definit tu ceva in store pentru goal-urile prietenilor?

Reply:
M-am gandit ca tot acolo se ajunge pt ca GoalList se folosesc de ele oricum si incarc in ele in functie de ce este dat